From 16387887ec29013e49c1db546f943b346540364a Mon Sep 17 00:00:00 2001
From: nhannguyen <nhan.nguyen.yb@renesas.com>
Date: Thu, 3 May 2018 12:03:52 +0700
Subject: [PATCH 284/504] ARM: dts: r8a7742-iwg21m: Disable QSPI to enable VIN2

There's a workaround in u-boot to control the selection of qspi and vin2
through GPIO0_18. And when working, vin2 should be selected as default
IF after kernel boot. This patch disables qspi and enables vin2.

Signed-off-by: nhannguyen <nhan.nguyen.yb@renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21m.dts | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21m.dts b/arch/arm/boot/dts/r8a7742-iwg21m.dts
index 730a0cf..193162e 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21m.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21m.dts
@@ -185,17 +185,6 @@
 	clock-frequency = <20000000>;
 };
 
-/* When “GP0_18” pin is set to low, QSPI is selected
- * When “GP0_18” pin is set to high, VIN2 is selected */
-&gpio0 {
-	enable-qspi{
-		gpio-hog;
-		gpios = <18 GPIO_ACTIVE_HIGH>;
-		output-low;
-		line-name = "enable-qspi";
-	};
-};
-
 &gpio1 {
 	can1-trx-en-gpio{
 		gpio-hog;
@@ -483,7 +472,7 @@
 	pinctrl-0 = <&qspi_pins>;
 	pinctrl-names = "default";
 
-	status = "okay"; /* multiplexed with VI2 */
+	status = "disabled"; /* multiplexed with VI2 */
 
 	flash: flash@0 {
 		#address-cells = <1>;
-- 
2.7.4

